docs/integration/alerts-googlechat-integration.md

```diff
@@ -0,0 +1,50 @@
+title: Google Chat Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Google Chat. Use them together to get the information about each notification quickly and on any device.
+
+## In Google Chat
+
+**1.** Create a new space that will be used to receive notifications from Sematext, or use an existing one.
+
+On the spaces tab, click the **+** icon:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Add Space" src="../../images/integrations/create-googlechat-integration-add.png" title="Create Google Chat Integration - Add Space">
+
+**2.** Select **Create Space** option:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - New Space" src="../../images/integrations/create-googlechat-integration-new-space.png" title="Create Google Chat Integration - New Space">
+
+**3.** Give your space a name and click **Create**:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Create Space" src="../../images/integrations/create-googlechat-integration-create.png" title="Create Google Chat Integration - Create Space">
+
+**4.** Open the drop down menu with your space's name and select **Manage webhooks**:
+
+<img class="content-modal-image" alt="Create Google Chat Integration - Manage Webhooks" src="../../images/integrations/create-googlechat-integration-manage-webhooks.png" title="Create Google Chat Integration - Manage Webhooks">
+
+**5.** Give your webhook a name (and optionally an avatar) and click **Save**:
```

Comment:
Is webhook the right term to use here?  Is that what they use?  Asking because we use "Webhook" for a generic type of "notify by calling some URL" sort of notification hook.

Reply:
The dialog in Google chat is called "Incoming Webhooks" and the field just says "Name", so I believe it's the right term to use here simply to match the UI. Do you think we should change it?